speculos/mcu/seproxyhal.py

```diff
@@ -270,7 +275,10 @@ def __init__(self,
                                                    self.socket_helper.wait_until_tick_is_processed)
         self.time_ticker_thread.start()
 
-        self.usb = usb.USB(self.socket_helper.queue_packet, transport=transport)
+        usb_transport = transport if transport in ['hid', 'u2f'] else 'hid'
```

Comment:
```suggestion
        usb_transport = transport if transport.lower() in ['hid', 'u2f'] else 'hid'
```

Reply:
Arguably weird to check that here, when `USB.__init__` already has equivalent check. Might as well replace the `raise ValueError` by a `self.transport = Hid(self.send_xfer)` there if fallbacking on HID is the expected behavior.